src/go/rpk/pkg/cli/cmd/topic_test.go

```diff
@@ -258,200 +258,200 @@ func TestDescribeTopic(t *testing.T) {
-  1               1                 [1]        [1]               
-  2               1                 [1]        [1]               
-  3               1                 [1]        [1]               
-  4               1                 [1]        [1]               
-  5               1                 [1]        [1]               
-  6               1                 [1]        [1]               
-  7               1                 [1]        [1]               
-  8               1                 [1]        [1]               
-  9               1                 [1]        [1]               
-  10              1                 [1]        [1]               
-  11              1                 [1]        [1]               
+			args: []string{"Cologne", "--page-size", "31", "--page", "4"},
+			expectedOutput: `  Name            Cologne
+  Internal        false
+  Cleanup policy  compact
+  Config:
+  Name            Value    Read-only  Sensitive
+  key             value    false      false
+  Partitions      1 - 12 out of 12
+  Partition       Leader            Replicas   In-Sync Replicas
+  0               1                 [1]        [1]
+  1               1                 [1]        [1]
+  2               1                 [1]        [1]
+  3               1                 [1]        [1]
+  4               1                 [1]        [1]
+  5               1                 [1]        [1]
+  6               1                 [1]        [1]
+  7               1                 [1]        [1]
+  8               1                 [1]        [1]
+  9               1                 [1]        [1]
+  10              1                 [1]        [1]
+  11              1                 [1]        [1]
 `,
 		},
 		{
-			name:	"it should show all the partitions if the page size is negative",
+			name: "it should show all the partitions if the page size is negative",
 			admin: &mocks.MockAdmin{
 				MockDescribeTopics: func(_ []string) ([]*sarama.TopicMetadata, error) {
 					return []*sarama.TopicMetadata{
 						{
-							Name:		"Cologne",
-							Partitions:	generatePartitions(5),
+							Name:       "Cologne",
+							Partitions: generatePartitions(5),
 						},
 					}, nil
 				},
 			},
-			args:	[]string{"Cologne", "--page", "-1", "--page-size", "4"},
-			expectedOutput: `  Name            Cologne  
-  Internal        false    
-  Cleanup policy  compact  
-  Config:         
-  Name            Value    Read-only  Sensitive  
-  key             value    false      false      
-  Partitions      1 - 5 out of 5  
-  Partition       Leader          Replicas   In-Sync Replicas  
-  0               1               [1]        [1]               
-  1               1               [1]        [1]               
-  2               1               [1]        [1]               
-  3               1               [1]        [1]               
-  4               1               [1]        [1]               
```

Comment:
This is problematic place where newest `crlfmt` is removing white spaces from the expected output which brake the unit test. The `tablewriter` from some reason produces white characters at the end of each line.

Reply:
Yup, these tests were a pain to write because of that haha.
But wait, so does this break the test? If crlfmt is trying to edit raw strings, that's a bug.